Header header

float64 CurrentSpeed                     # m/s
float64 TargetSpeed                      # m/s
float64 SpeedError                       # m/s
float64 AccelerationCmd                  # m/ss
float64 Curvature                        # rad
float64 LatError                         # m
float64 DirError                         # rad
float64 SteeringAngleCmd                 # rad
float64 CurvatureContribution            # rad
float64 LatContribution                  # rad
float64 DirContribution                  # rad
geometry_msgs/Pose CurrentPose